packages/date-time/src/components/Calendar/Calendar.base.tsx

```diff
@@ -0,0 +1,343 @@
+      showMonthPickerAsOverlay: showMonthPickerAsOverlay,
+      overlayedWithButton: overlayedWithButton,
+      showGoToToday: showGoToToday,
+      showWeekNumbers: showWeekNumbers
+    });
+
+    return (
+      <div
+        className={css(rootClass, classes.root, className, 'ms-slideDownIn10')}
+        role="application"
+        onKeyDown={this._onDatePickerPopupKeyDown}
+      >
+        {isDayPickerVisible && (
+          <CalendarDay
+            selectedDate={selectedDate!}
+            navigatedDate={navigatedDayDate!}
+            today={this.props.today}
+            onSelectDate={this._onSelectDate}
+            onNavigateDate={this._onNavigateDayDate}
+            onDismiss={this.props.onDismiss}
+            firstDayOfWeek={firstDayOfWeek!}
+            dateRangeType={dateRangeType!}
+            strings={strings!}
+            onHeaderSelect={onHeaderSelect}
+            navigationIcons={navigationIcons!}
+            showWeekNumbers={this.props.showWeekNumbers}
+            firstWeekOfYear={this.props.firstWeekOfYear!}
+            dateTimeFormatter={this.props.dateTimeFormatter!}
+            showSixWeeksByDefault={this.props.showSixWeeksByDefault}
+            minDate={minDate}
+            maxDate={maxDate}
+            workWeekDays={this.props.workWeekDays}
+            componentRef={this._dayPicker}
+            showCloseButton={showCloseButton}
+            allFocusable={allFocusable}
+          />
+        )}
+        {isDayPickerVisible && isMonthPickerVisible && <div className={classes.divider} />}
+        {isMonthPickerVisible ? (
+          <div className={classes.monthPickerWrapper}>
+            <CalendarMonth
+              navigatedDate={navigatedMonthDate!}
+              selectedDate={navigatedDayDate!}
+              strings={strings!}
+              onNavigateDate={this._onNavigateMonthDate}
+              today={this.props.today}
+              highlightCurrentMonth={highlightCurrentMonth!}
+              highlightSelectedMonth={highlightSelectedMonth!}
+              onHeaderSelect={onHeaderSelect}
+              navigationIcons={navigationIcons!}
+              dateTimeFormatter={this.props.dateTimeFormatter!}
+              minDate={minDate}
+              maxDate={maxDate}
+              componentRef={this._monthPicker}
+            />
+            {this._renderGoToTodayButton(classes)}
+          </div>
+        ) : (
+          this._renderGoToTodayButton(classes)
+        )}
+      </div>
+    );
+  }
+
+  public focus(): void {
+    if (this.state.isDayPickerVisible && this._dayPicker.current) {
+      this._dayPicker.current.focus();
+    } else if (this.state.isMonthPickerVisible && this._monthPicker.current) {
+      this._monthPicker.current.focus();
+    }
+  }
+
+  private _renderGoToTodayButton = (classes: IProcessedStyleSet<ICalendarStyles>) => {
+    const { showGoToToday, strings } = this.props;
+    return (
+      showGoToToday && (
+        <button
```

Comment:
Do we prefer HTML buttons in general here?

Reply:
yes and no. Originally I added these as DefaultButtons and overrode the styling, but advice from dzearing suggested we do these as standard buttons for now while the Button control gets refactored